portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -501,22 +501,9 @@ definition {
 
 	macro editTextBodyToolBar {
 		SelectFrame.selectFrameNoLoading(locator1 = "CKEditor#BODY_FIELD_IFRAME");
-
```

Comment:
@gleicelisbino, let's keep this blank line to separate the instructions. :wink:

Reply:
@natocesarrego Okay!